### PR TITLE
Proposing a .gitignore file

### LINKED ISSUE #51: Missing .gitignore
Should we add a gitignore file for ignoring some files/folders?
eg. .vscode folder that is created when creating a local launch.json configuration

### PR DESCRIPTION
**Pull request by cerebrux**
The logic behind a .gitignore is to add some files
that will be produced by tools, builds, IDEs etc.
These files usually end up in our local git repo and
clutter our git status message.

This commit fixes #51

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+*.dylib
+*-amd64
+*-386
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE, GoLand, VScode
+*.out
+.idea/
+.vscode/
```
Comment (cerebrux) [inline .gitignore]: @drpaneas are these enough for start ? If yes, remove WIP and merge it, per favor.

Comment (drpaneas) [reply .gitignore]: what all those? AFAIK only `.vscode` is the one we use, right?

Comment (cerebrux) [reply .gitignore]: Yes you are right, I will remove the rest of them for now and we will add them as we go <--- <see what I did here :) ?>

**Review by drpaneas — CHANGES_REQUESTED** [review]
LGTM for now, we need to keep this updated for the future.

**Review by drpaneas — APPROVED** [review]

**Pull request merged**